transforms/universal/web2parquet/dpk_web2parquet/utils.py

```diff
@@ -0,0 +1,27 @@
+# (C) Copyright IBM Corp. 2024.
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+from datetime import datetime
+
+def get_file_info(headers, url):
```

Comment:
It doesn't look this function is used anywhere. Can we call this in the `transform.py` to make the code cleaner?